src/components/MyKiva/MyKivaProfile.vue

```diff
@@ -40,16 +40,20 @@ const props = defineProps({
 		type: Object,
 		default: () => ({}),
 	},
+	userInfo: {
+		type: Object,
+		default: () => ({}),
+	},
 	isLoading: {
 		type: Boolean,
 		default: false,
 	},
 });
 
-const { lender } = toRefs(props);
+const { lender, userInfo } = toRefs(props);
```

Comment:
For a guest in this situation, I'm assuming the `lender` image doesn't exist? Does the avatar show the first/last name initials like expected?

Reply:
If the avatar exists with initials, all is well (so I'll approve the PR for now), but if the initials are missing, please investigate.